src/dispatch/plugins/dispatch_slack/feedback/interactive.py

```diff
@@ -29,20 +30,35 @@
 )
 
 from .enums import (
-    FeedbackNotificationActionIds,
-    FeedbackNotificationActions,
-    FeedbackNotificationBlockIds,
+    IncidentFeedbackNotificationActionIds,
+    IncidentFeedbackNotificationActions,
+    IncidentFeedbackNotificationBlockIds,
+    OncallShiftFeedbackNotificationActionIds,
+    OncallShiftFeedbackNotificationActions,
+    OncallShiftFeedbackNotificationBlockIds,
 )
 
 
+class OncallShiftFeedbackRating(DispatchEnum):
+    no_effort = "No effort"
+    little_effort = "Little effort"
+    moderate_effort = "Moderate effort"
+    lots_of_effort = "Lots of effort"
+    very_high_effort = "Very high effort"
+    all_out_effort = "All out effort"
```

Comment:
Trying to align the adjectives on the scale ("little, moderate, lots, very high, extreme") to the extent we can. But there is something missing here -- the visual element. Is there any way that Slack would allow us to do a [Visual Analogue Scale](https://en.wikipedia.org/wiki/Visual_analogue_scale)?